azkaban-common/src/main/java/azkaban/project/NodeBeanLoader.java

```diff
@@ -86,15 +90,84 @@ public AzkabanNode toAzkabanNode(final NodeBean nodeBean) {
     }
   }
 
+  private void validateSchedule(final FlowTriggerBean flowTriggerBean) {
+    final Map<String, String> scheduleMap = flowTriggerBean.getSchedule();
+
+    Preconditions.checkNotNull(scheduleMap, "flow trigger schedule must not be null");
+
+    Preconditions.checkArgument(scheduleMap.containsKey("type") && scheduleMap.get("type")
+        .equals("cron"), "flow trigger schedule type must be cron");
```

Comment:
"type", "cron" and "value" are already defined in Constants.java. You can use them directly.

Reply:
thanks, updated